<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CJumpStepDlg</class>
 <widget class="QDialog" name="CJumpStepDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JUMP STEP</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MainUi.qrc">
    <normaloff>:/icon/icon/recursive.ico</normaloff>:/icon/icon/recursive.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #2b579a;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>JUMP STEP </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmStepControl">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeItems">
        <column>
         <property name="text">
          <string>STEP INFO</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>STEP NO</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>STEP_NAME_ORG</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnJump">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>JUMP</string>
          </property>
          <property name="icon">
           <iconset resource="../MainUi.qrc">
            <normaloff>:/icon/icon/recursive.ico</normaloff>:/icon/icon/recursive.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>CANCEL</string>
          </property>
          <property name="icon">
           <iconset resource="../MainUi.qrc">
            <normaloff>:/icon/icon/cancel.ico</normaloff>:/icon/icon/cancel.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="lblLoopCount">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>TEST LOOP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinLoopCount">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkBurnIn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>BRUN-IN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leBurnIn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="lblStepNo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>STEP NO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leStepNo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chkOnlySelect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>ONLY SELECT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkChamber">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>CHAMBER</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../MainUi.qrc"/>
 </resources>
 <connections/>
</ui>
